internal/cli/apps.go

```diff
@@ -868,48 +868,37 @@ func stringSliceToPtr(s []string) *[]string {
 	return &s
 }
 
-func (c *cli) appPickerOptions() (pickerOptions, error) {
-	list, err := c.api.Client.List()
-	if err != nil {
-		return nil, err
-	}
-
-	tenant, err := c.getTenant()
-	if err != nil {
-		return nil, err
-	}
-
-	// NOTE(cyx): To keep the contract for this simple, we'll rely on the
-	// implicit knowledge that the default value for the picker is the
-	// first option. With that in mind, we'll use the state in
-	// tenant.DefaultAppID to determine which should be chosen as the
-	// default.
-	var (
-		priorityOpts, opts pickerOptions
-	)
-	for _, c := range list.Clients {
-		// Empty type means the default client that we shouldn't display.
-		// TODO(sergiught): We only need to exclude generic app types for
-		// the auth0 qs download command. Fix this in separate PR.
-		if c.GetAppType() == "" {
-			continue
+func (c *cli) appPickerOptions(requestOpts ...management.RequestOption) pickerOptionsFunc {
+	requestOpts = append(requestOpts, management.Parameter("is_global", "false"))
+
+	return func() (pickerOptions, error) {
+		clientList, err := c.api.Client.List(requestOpts...)
+		if err != nil {
+			return nil, err
+		}
+
+		tenant, err := c.getTenant()
+		if err != nil {
+			return nil, err
 		}
 
-		value := c.GetClientID()
-		label := fmt.Sprintf("%s %s", c.GetName(), ansi.Faint("("+value+")"))
-		opt := pickerOption{value: value, label: label}
+		var priorityOpts, opts pickerOptions
+		for _, client := range clientList.Clients {
+			value := client.GetClientID()
+			label := fmt.Sprintf("%s %s", client.GetName(), ansi.Faint("("+value+")"))
+			option := pickerOption{value: value, label: label}
 
-		// check if this is currently the default application.
-		if tenant.DefaultAppID == c.GetClientID() {
-			priorityOpts = append(priorityOpts, opt)
-		} else {
-			opts = append(opts, opt)
+			if tenant.DefaultAppID == client.GetClientID() {
```

Comment:
🤔 There's very minimal usage of the default app id at the moment within the cli, could be something we might want to remove or enhance support for.

Reply:
I think it's fine to leave it as it is for now.